README.md

```diff
@@ -505,62 +504,61 @@ Lisp                                 2              0              3
 -----------------------------------------------------------------------------------
 SUM:                              3434         176974         243934         903874
 -----------------------------------------------------------------------------------
-
-</pre>
+```
 
 To run cloc on Windows computers, one must first open up a command (aka DOS) window and invoke cloc.exe from the command line there.
 [](1}}})
 <a name="Options"></a> []({{{1)
 # [Options &#9650;](#___top "click to go to top of document")
 
-<pre>
+```
 prompt> cloc --help
 
-Usage: cloc [options] &lt;file(s)/dir(s)/git hash(es)&gt; | &lt;set 1&gt; &lt;set 2&gt; | &lt;report files&gt;
+Usage: cloc [options] <file(s)/dir(s)/git hash(es)> | <set 1> <set 2> | <report files>
 
  Count, or compute differences of, physical lines of source code in the
  given files (may be archives such as compressed tarballs or zip files,
  or git commit hashes or branch names) and/or recursively below the
  given directories.
 
  Input Options
-   --extract-with=&lt;cmd&gt;      This option is only needed if cloc is unable
+   --extract-with=<cmd>      This option is only needed if cloc is unable
                              to figure out how to extract the contents of
                              the input file(s) by itself.
-                             Use &lt;cmd&gt; to extract binary archive files (e.g.:
-                             .tar.gz, .zip, .Z).  Use the literal '&gt;FILE&lt;' as
+                             Use <cmd> to extract binary archive files (e.g.:
+                             .tar.gz, .zip, .Z).  Use the literal '>FILE<' as
```

Comment:
Should >FILE< be replaced with <FILE> ?

Reply:
The README is correct as-is with regards to ``<cmd>`` and ``>FILE<``.  I have the cited block of text between HTML ``<pre>`` and ``</pre>`` tags so the ampersand groups are correct.

In retrospect my choice of >FILE< as a literal wasn't the best idea, but no sense changing it now.

Reply:
OK. I assumed there were some reason.